src/element/bounds.test.ts

```diff
@@ -1,9 +1,23 @@
-import { getElementAbsoluteCoords } from "./bounds";
-import { ExcalidrawElement } from "./types";
+import { getElementAbsoluteCoords, getElementBounds } from "./bounds";
+import { ExcalidrawElement, ExcalidrawLinearElement } from "./types";
 
-const _ce = ({ x, y, w, h }: { x: number; y: number; w: number; h: number }) =>
+const _ce = ({
+  x,
+  y,
+  w,
+  h,
+  a,
+  t,
+}: {
+  x: number;
+  y: number;
+  w: number;
+  h: number;
+  a?: number;
+  t?: string;
```

Comment:
can we use proper names?, width, height, etc..

Reply:
Same here. The purpose is to use the short names, I suppose.
But, I don't have strong opinion with this. It was just there.

Reply:
We have space.. let's be more descriptive.

Reply:
That function is a "helper" function and its whole purpose I guess was to avoid having the full names. If we are going to remove these short variable names, we could as well remove that function altogether. It's fine by me but it's unrelated to this PR.